packages/core/src/CheckBox/index.tsx

```diff
@@ -77,15 +87,18 @@ const CheckBox: React.FC<CheckBoxProps> = ({ checked = false, onChange }) => {
   };
 
   return (
-    <label className={classes.contain}>
-      <input
-        data-testid="checkbox"
-        type="checkbox"
-        checked={isChecked}
-        onChange={handleChange}
-      />
-      <span className={classes.checkmark} />
-    </label>
+    <>
+      <label className={classes.contain} htmlFor={props.name}>
```

Comment:
Should this outer `label` be a `div`?

Reply:
no, it needs it for the checkbox ability, apparently